src/test/java/fi/helsinki/cs/tmc/core/communication/SubmissionPollerTest.java

```diff
@@ -17,15 +18,14 @@
 
 import org.mockito.Mockito;
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
```

Comment:
Import order (I know most of this is existing code, but it'd be much appreciated if you'd fix all the imports in this file :))